src/components/SidePanel/SidePanel.less

```diff
@@ -90,7 +91,9 @@
 						display: flex;
 						justify-content: center;
 						align-items: center;
-
+						padding: 8px;
+						margin: 12px;
```

Comment:
Make this `@size-standard` which is defined as `12px` in our [variables.less file](https://github.com/appnexus/lucid/blob/f5a89a2033935b52cf1f79966072461d9beeb333/src/styles/variables.less#L29).